core/common/src/main/java/alluxio/security/authorization/AccessControlList.java

```diff
@@ -295,6 +298,43 @@ public AclActions getPermission(String user, List<String> groups) {
     return new AclActions(mOtherActions);
   }
 
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof AccessControlList)) {
+      return false;
+    }
+    AccessControlList that = (AccessControlList) o;
+    return mOwningUser.equals(that.mOwningUser)
+        && mOwningGroup.equals(that.mOwningGroup)
+        && mUserActions.equals(that.mUserActions)
+        && mGroupActions.equals(that.mGroupActions)
+        && mMaskActions.equals(that.mMaskActions)
+        && mOtherActions.equals(that.mOtherActions);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hashCode(mOwningUser, mOwningGroup, mUserActions, mGroupActions, mMaskActions,
+        mOtherActions);
+  }
+
+  @Override
+  public String toString() {
```

Comment:
Strange that `toString` uses `toCliString`?

Reply:
I just removed this method.